BackofficeBundle/Resources/public/coffee/table/TableviewCollectionView.coffee

```diff
@@ -221,6 +222,7 @@ TableviewCollectionView = OrchestraView.extend(
         else if $.isPlainObject(conf.data)
           $.extend(request, conf.data)
 
+        $.ajaxSetup().abortXhr()
```

Comment:
I create a revert https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1196
We'll talk at the next ceremony